package.json

```diff
@@ -94,9 +93,9 @@
     "zone.js": "^0.8.14"
   },
   "dependencies": {
-    "@types/fullcalendar": "3.5.2",
+    "@types/fullcalendar": "^3.8.0",
```

Comment:
@types/fullcalendar@3.8.0 is now deprecated. FullCalendar provides its own type definition.
This line should be removed